Cargo.toml

```diff
@@ -10,6 +10,13 @@ description = "UNIC - Unicode and Internationalization Crates"
 categories = ["parsing", "rendering", "encoding", "development-tools"]
 readme = "README.md"
 
+exclude = [
+    "data/**/*.txt",
+    "tools/**/*.py",
+    ".cargo/*.sh",
+    "tests/*conformance*.rs",
```

Comment:
Huhu. I didn't think of doing it like this. This allows keeping the `tests/unicode_version.rs` ones. Nice! (And we should document it, also.)